### PR TITLE
Collection template support to List of String, Integer and Number 

### PR DESCRIPTION
**Pull request by amsidhlokhande**

### CODE REVIEW

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml [hunk 1]
```diff
@@ -8,13 +8,13 @@
   
   <table><thead><tr>
   
-    <td ><b>Isin</b></td>
+    <td style="text-align:center; width:10%" ><b>Isin</b></td>
```
Comment (robmoffat) [inline libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml:11]: ISIN is a string, it shouldn't be centered.

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml [hunk 2]
```diff
@@ -8,13 +8,13 @@
...
   
     <td style="text-align:center;" ><b>Bid Axed</b></td>
   
     <td style="text-align:center;" ><b>Ask Axed</b></td>
   
-    <td ><b>Creator</b></td>
+    <td style="text-align:center; width:10%" ><b>Creator</b></td>
```
Comment (robmoffat) [inline libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml:17]: same here

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testCollectionEditMessage.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+  "formdata" : {
+    "type" : "org.finos.symphony.toolkit.workflow.fixture.person",
+    "names" : [ "abc", "pqr" ],
+    "addresses" : [ {
+      "city" : "Pune"
```
Comment (robmoffat) [inline libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testCollectionEditMessage.json:6]: This isn't testing a primitive?  Don't we already have a test for beans like Address?

File: tools/koreai/src/main/java/org/finos/symphony/toolkit/koreai/request/KoreAIRequesterImpl.java [hunk 4]
```diff
@@ -34,26 +38,37 @@
     protected JsonNodeFactory jnf;
     protected String jwt;
     protected String url;
+    protected TrustManagerFactory tmf;
```
Comment (robmoffat) [inline tools/koreai/src/main/java/org/finos/symphony/toolkit/koreai/request/KoreAIRequesterImpl.java:41]: How has this ended up here?  It doesn't seem to be related to your change

**Comment by robmoffat** [comment]
ok, I see somehow we built this on top of my Kore AI changes.  Is there any way we can exclude those commits?  In retrospect, they're not required/sensible.

**Comment by robmoffat** [comment]
Specifically, 768d2aa.  Can we remove this from the PR?

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml (hunk 1 above)
Comment (amsidhlokhande) [reply libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml]: Done. Kept the String representation as like older one

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml (hunk 2 above)
Comment (amsidhlokhande) [reply libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testAxesTableEditMessageML.ml]: Done. Kept the String representation as like older one

File: libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testCollectionEditMessage.json (hunk 3 above)
Comment (amsidhlokhande) [reply libs/chat-workflow-spring-boot-starter/src/test/resources/org/finos/symphony/toolkit/workflow/testCollectionEditMessage.json:6]: Yes we have the test for beans like address. But we are checking is there any impact of beans with adding support for primitive. So this work class contains List<String> and List<Address>. I don't think so there is any issue with testing beans along with primitive

File: tools/koreai/src/main/java/org/finos/symphony/toolkit/koreai/request/KoreAIRequesterImpl.java (hunk 4 above)
Comment (amsidhlokhande) [reply tools/koreai/src/main/java/org/finos/symphony/toolkit/koreai/request/KoreAIRequesterImpl.java:41]: I have not done any changes. Even git history also not showing any change on my name

**Review by robmoffat — APPROVED** [review]
